extension/scripts/background.js

```diff
@@ -915,8 +915,11 @@ async function notifyUser(title, message, url) {
 		};
 
 	if (settings.notifications.tts) {
-		window.speechSynthesis.speak(new SpeechSynthesisUtterance(title));
-		window.speechSynthesis.speak(new SpeechSynthesisUtterance(message));
+    	const ttsTitle = new SpeechSynthesisUtterance(title);
+    	const ttsMessage = new SpeechSynthesisUtterance(message);
+    	ttsTitle.volume = ttsMessage.volume = settings.notifications.volume / 100;
```

Comment:
I do know that this is valid syntax, but not sure how much I like it.

Reply:
Agreed, spread it on 2 lines, also replaced spaces with tabs